src/TodoListItem.js

```diff
@@ -0,0 +1,12 @@
+function TodoListItem(props) {
```

Comment:
note that you can deconstruct props here too .. and just use 

function TodoListItem({todo})

and use it below ..